### ISSUE TITLE
Query: Can we move the code base to from master branch to develop branch?

### ISSUE THREAD

**Issue by madguy02** [Rajan/lesspod#11]
Moving the project from master to develop will help at release cycle, where the freezed features will live on master branch.

Its not an urgency, but required step

**Comment by madguy02** [comment]
@Rajan  your thoughts on this?

**Comment by Rajan** [comment]
Yes. The question is what is the ideal time to do it? In a week?

**Comment by s-kris** [comment]
We can do that once we have some usable release.

**Comment by madguy02** [comment]
yeah, after an usable release is fine, keeping the thread open as of now

**Comment by sujaykundu777** [comment]
Yes. I really agree on making a separate branch for development release. But for now let's have our first release on master.
